### DISCUSSION TITLE
Allow swtiching between HPaned and VPaned in ImageGeneratorDialog.

### DISCUSSION THREAD

**Discussion by k4nzdroid** [zim-desktop-wiki/zim-desktop-wiki#2404] [Ideas]
Currently, ImageGeneratorDialog has a VPaned to display ImageWindow and SourceWindow. 

Sometime, It is more reasonable to use an HPaned to do that. For example, the source has quiet a lot of short lines.

Doing that is to make maximum utilization of the screen space.

**Comment by jaap-karssenberg** [comment]
Sure, if we could fit a "rotate" icon somewhere in the screen and switch
views that would be great. Using a preference would also be OK, but less
flexible to switch depending on content.

--- Jaap
